### ISSUE TITLE
Add a contributors guide

### ISSUE THREAD

**Issue by kevmoo** [Dartwatch/builtwithdart.com#3]
How to get the site running on a dev machine w/ the App Engine Launcher, etc.
Guidelines, etc.

**Comment by chrisbu** [comment]
Good idea - Contributors guide is now available via the /contact/ page.  Still todo: add guidelines.
